rke/templates/metrics_v0.5.1.go

```diff
@@ -0,0 +1,237 @@
+package templates
```

Comment:
The chosen file name is confusing as it makes impression that it's for metrics server 0.5.1.
But in fact we've been using `rancher/mirrored-metrics-server:v0.5.1` with `metrics_v0.5.0.go` template for a while now.
Maybe name it (and related variables) as `metrics_v0.5.1-fix.go` to help avoid confusion? @kinarashah , please chime in on this.

Reply:
Sure. Will be updating it.